addons/mail/static/src/discuss/call/common/call_action_list.xml

```diff
@@ -30,30 +30,63 @@
-                    aria-label="Reject"
-                    title="Reject"
-                    t-att-disabled="rtc.state.hasPendingRequest"
-                    t-on-click="onClickRejectCall">
-                    <i class="oi oi-close fa-fw m-2" t-att-class="{ 'fa-lg': !isSmall }"/>
-                </button>
                 <button t-if="props.thread.selfMember?.rtc_inviting_session_id?.is_camera_on" class="btn smaller btn-success d-flex m-1 border-0 rounded-circle shadow-none align-items-center p-0"
                     aria-label="Accept with camera"
                     title="Accept with camera"
                     t-att-disabled="rtc.state.hasPendingRequest"
                     t-on-click="(ev) => this.onClickToggleAudioCall(ev, { camera: true })">
                     <i class="fa fa-video-camera fa-fw m-2" t-att-class="{ 'fa-lg': !isSmall }"/>
                 </button>
-                <t t-if="props.thread.eq(rtc.channel)" t-set="callText">Disconnect</t>
-                <t t-else="" t-set="callText">Join Call</t>
-                <button class="btn smaller d-flex m-1 border-0 rounded-circle shadow-none align-items-center p-0"
-                    t-att-aria-label="callText"
-                    t-att-class="{ 'btn-danger': isOfActiveCall, 'btn-success': !isOfActiveCall }"
-                    t-att-disabled="rtc.state.hasPendingRequest"
-                    t-att-title="callText"
-                    t-on-click="onClickToggleAudioCall">
-                    <i class="fa fa-phone fa-fw m-2" t-att-class="{ 'fa-lg': !isSmall }"/>
-                </button>
+                <t t-if="!isOfActiveCall and props.thread.useCameraByDefault !== null">
+                    <button t-if="props.thread.selfMember?.rtc_inviting_session_id"
+                        class="btn btn-danger smaller d-flex gap-1 m-1 border-0 align-items-center"
+                        t-att-disabled="rtc.state.hasPendingRequest"
+                        aria-label="Reject"
+                        title="Reject"
+                        t-on-click="onClickRejectCall">
+                        Reject
+                    </button>
+                    <t t-set="joinBackText" t-if="props.thread.useCameraByDefault">Join Video Call</t>
+                    <t t-else="" t-set="joinBackText">Join Call</t>
+                    <button
+                        class="btn btn-success smaller d-flex gap-1 m-1 border-0 align-items-center"
+                        t-att-disabled="rtc.state.hasPendingRequest"
+                        t-att-aria-label="joinBackText"
+                        t-att-title="joinBackText"
+                        t-on-click="(ev) => this.onClickToggleAudioCall(ev, { camera: props.thread.useCameraByDefault })"
+                    >
+                        <i class="fa fa-fw" t-att-class="{'fa-video-camera': props.thread.useCameraByDefault, 'fa-phone': !props.thread.useCameraByDefault}"/>Join
+                    </button>
+                </t>
```

Comment:
`.smaller` classname is combined with `.fa-lg` icons in `!ui.isSmall`.

Without this, the size of icon is smaller in discuss app as the call action buttons, which is not good:

<img width="2264" height="257" alt="Screenshot 2025-08-13 at 12 55 30" src="https://github.com/user-attachments/assets/6b757ef2-0386-4f65-9308-42e1136b23da" />

Call action is spaghetti code, ideally should use `ActionList` and have consistent style solved once and for all. I'm working on it, but here in the meantime you could have `.smaller` conditioned on `isSmall`.

Reply:
Would also `rounded-3` the squared icon so it looks more in line with the rounded call actions.

This will eventually be changed positively by converting them to `ActionList` I'm working on, but better consider making these small changes while there's not the pass in using `ActionList` yet.